core/runtime/src/main/java/io/quarkus/runtime/logging/ColorPatternFormatterWithBanner.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.runtime.logging;
+
+import java.util.logging.Handler;
+
+import org.jboss.logmanager.formatters.ColorPatternFormatter;
+
+public class ColorPatternFormatterWithBanner extends ColorPatternFormatter {
```

Comment:
Rather than having two formatter subclasses, what about having a single formatter called `BannerFormatter` which delegates to another `ExtFormatter`?

Reply:
I have extended ColorPatternFormatter, sinxe ConsolehandlerTest asserts for PatternFormatter instance only. In the single BannerFormatter , I have overriden  methods which were different in Pattern and ColorPattern and delegated  them to corresponding formatter instances